rocketpy/simulation/monte_carlo.py

```diff
@@ -359,6 +371,22 @@ def __export_flight_data(
             for export_item in self.export_list
         }
 
+        if self.export_function is not None:
+            try:
+                additional_exports = self.export_function(flight)
+            except Exception as e:
+                raise ValueError(
+                    "An error was encountered running your custom export function. "
+                    "Check for errors in 'export_function' definition."
+                ) from e
+
+            for key in additional_exports.keys():
+                if key in self.export_list:
+                    raise ValueError(
+                        f"Invalid export function, returns dict which overwrites key, '{key}'"
+                    )
```

Comment:
This `Exception` message could be much clearer. I believe it would be interesting to specify that a key from the `export_list` conflicts with a key from the `export_function` dictionary. Then the user know exactly which parameters to look.